lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -514,6 +514,47 @@ struct ExprVisitor {
     return result;
   }
 
+  // Handle conditional operator `?:`.
+  Value visit(const slang::ast::ConditionalExpression &expr) {
+    auto type = context.convertType(*expr.type);
+
+    // Handle condition.
+    Value cond = convertToSimpleBitVector(
+        context.convertExpression(*expr.conditions.begin()->expr));
+    cond = convertToBool(cond);
+    if (!cond)
+      return {};
+    auto conditionalOp = builder.create<moore::ConditionalOp>(loc, type, cond);
+
+    // Create blocks for true region and false region.
+    conditionalOp.getTrueRegion().emplaceBlock();
+    conditionalOp.getFalseRegion().emplaceBlock();
+
+    OpBuilder::InsertionGuard g(builder);
+
+    // Handle left expression.
+    builder.setInsertionPointToStart(conditionalOp.getBody(0));
+    auto trueValue = context.convertExpression(expr.left());
+    if (!trueValue)
+      // When `trueValue` is empty (e.g. expression is X or Z for now) , use
+      // builder without value to avoid errors.
+      builder.create<moore::YieldOp>(loc);
+    else
+      builder.create<moore::YieldOp>(loc, convertToSimpleBitVector(trueValue));
+
```

Comment:
`convertToSimpleBitVector()` may return `{}` if the type of trueValue isn't SBVT(Like `logic [0:5][2:0] p3`).